system/Common.php

```diff
@@ -563,6 +563,27 @@ function function_usable(string $functionName): bool
     }
 }
 
+if (! function_exists('selfclosing_html')) {
+    /**
+     * Endline HTML Tag
+     *
+     * Generates endline html tag by document type
+     *
+     * @internal
+     */
+    function selfclosing_html(): string
+    {
+        $config  = config('DocTypes');
+        $doctype = $config->xhtml;
```

Comment:
Please add the logic when `$config->xhtml` does not exist for uses who do not update the config file.

Reply:
Done